### PR TITLE
CFE-2952: Fix example output in select_region.cf

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a pull request! Maybe @olehermanse can review this?

**Comment by nickanderson** [comment]
@cf-bottom jenkins, please

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/2100/)

https://ci.cfengine.com/job/pr-pipeline/2100/

**Comment by nickanderson** [comment]
@olehermanse does this look OK to you? The failure in Jenkins seems unrelated.

**Comment by olehermanse** [comment]
@nickanderson Ticket seems unrelated? https://tracker.mender.io/browse/CFE-2952

File: examples/select_region.cf [hunk 1]
```diff
@@ -44,6 +44,7 @@ bundle agent main
                   MATCHING inside the region described by regular expressions
                   matching the beginning and end of the region.",
         create    => "true",
+        #action => warn_or_fix( "warn" ),
```
Comment (olehermanse) [inline examples/select_region.cf]: I don't understand, this is only a comment. What is the change here?

**Comment by nickanderson** [comment]
> @nickanderson Ticket seems unrelated? https://tracker.mender.io/browse/CFE-2952

I started this when I was working that ticket to improve docs. This change is to make the example run as part of the example tests.

File: examples/select_region.cf (hunk 1 above)
Comment (nickanderson) [reply examples/select_region.cf]: I have re-visited this when time allowed over a long period, it was just the last change that I was making, all the stuff is squashed now.

**Comment by olehermanse** [comment]
> I started this when I was working that ticket to improve docs. This change is to make the example run as part of the example tests.

Why wouldn't it run before? Why does this macro change make a difference?

**Pull request closed without merging**